src/client/signatures/canonical_json/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+//! Remote-Settings Canonical JSON library to serialize JSON to String
```

Comment:
Link to the canonical JSON spec, please.  And explain what canonical JSON is.  The Remote-Settings piece is less important -- say that last, since it's just colour for other consumers.

In fact, while looking around, I found https://github.com/zmanian/canonical_json -- what are the pros and cons of this implementation?

Reply:
it does not support floating-point numbers, cannot convert Unicode characters to Unicode escape sequences

Reply:
Ah!  Then it's definitely worth calling out how this compares to that implementation in the README or similar.

Reply:
yes